core/src/main/java/org/elasticsearch/index/analysis/AnalysisService.java

```diff
@@ -298,10 +300,6 @@ public NamedAnalyzer analyzer(String name) {
         return analyzers.get(name);
     }
 
-    public NamedAnalyzer defaultAnalyzer() {
-        return defaultAnalyzer;
-    }
-
     public NamedAnalyzer defaultIndexAnalyzer() {
```

Comment:
I wanted to be consistent with MappedFieldType, which takes an `analyzer` and a `search_analyzer` as settings but still exposes an `indexAnalyzer()` and a `searchAnalyzer()` in its API